src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/SessionPass.php

```diff
@@ -26,20 +26,22 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        // BC layer: Make "session" an alias of ".session.do-not-use" when not overriden by the user
+        // BC layer: Make "session" an alias of ".session.do-not-use" when not overridden by the user
         if (!$container->has('session')) {
             $alias = $container->setAlias('session', '.session.do-not-use');
-            $alias->setDeprecated('symfony/framework-bundle', '5.3', 'The "%alias_id%" service is deprecated, use "$requestStack->getSession()" instead.');
+            $alias->setDeprecated('symfony/framework-bundle', '5.3', 'The "%alias_id%" service and "SessionInterface" alias are deprecated, use "$requestStack->getSession()" instead.');
+            // restore previous behavior
+            $alias->setPublic(true);
 
             return;
         }
 
         if ($container->hasDefinition('session')) {
             $definition = $container->getDefinition('session');
-            $definition->setDeprecated('symfony/framework-bundle', '5.3', 'The "%service_id%" service is deprecated, use "$requestStack->getSession()" instead.');
+            $definition->setDeprecated('symfony/framework-bundle', '5.3', 'The "%service_id%" service and "SessionInterface" alias are deprecated, use "$requestStack->getSession()" instead.');
         } else {
             $alias = $container->getAlias('session');
-            $alias->setDeprecated('symfony/framework-bundle', '5.3', 'The "%alias_id%" alias is deprecated, use "$requestStack->getSession()" instead.');
+            $alias->setDeprecated('symfony/framework-bundle', '5.3', 'The "%alias_id%" and "SessionInterface" aliases are deprecated, use "$requestStack->getSession()" instead.');
```

Comment:
`aliases` -> `alias` like in the other previous message?

Reply:
This one is about `the xxx and xxx aliases` while the previous messages are about `the xxx service and the xxx alias`, this looks correct to me

Reply:
You are right! I misread this. Thanks!